errors.toml

```diff
@@ -351,11 +351,6 @@ error = '''
 A primary key index cannot be invisible
 '''
 
-["ddl:3753"]
-error = '''
-Cannot create an expression index on a function that returns a JSON or GEOMETRY value
```

Comment:
why remove it here?

Reply:
After I changing it to internal, `make errdoc` will do it.